dev/SwipeControl/SwipeControl.h

```diff
@@ -179,8 +177,8 @@ class SwipeControl :
     tracker_ref<winrt::IInspectable> m_onPointerPressedEventHandler{ this };
 
     // Used on platforms where we have XamlRoot.
-    tracker_ref<winrt::IInspectable> m_onXamlRootPointerPressedEventHandler{ this };
-    tracker_ref<winrt::IInspectable> m_onXamlRootKeyDownEventHandler{ this };
+    RoutedEventHandler_revoker m_onXamlRootPointerPressedEventHandler{};
```

Comment:
The convention we use is m_xamlRootPointerPressedRevoker.

Reply:
thanks, copy paste error